packages/remix-react/browser.tsx

```diff
@@ -414,3 +464,75 @@ export function RemixBrowser(_props: RemixBrowserProps): ReactElement {
     </>
   );
 }
+
+function addPropRoutesToRemix(route: DataRouteObject, parentId: string) {
+  if (!window.__remixManifest.routes[route.id]) {
+    window.__remixManifest.routes[route.id] = {
+      index: route.index,
+      caseSensitive: route.caseSensitive,
+      id: route.id,
+      path: route.path,
+      parentId,
+      hasAction: false,
+      hasLoader: false,
+      hasClientAction: route.action != null,
+      hasClientLoader: route.loader != null,
+      hasErrorBoundary: route.hasErrorBoundary === true,
+      imports: [],
+      css: [],
+      module: undefined,
```

Comment:
These don't have modules to fetch, so that required some changes elsewhere to make `manifestRoute.module` an optional field